crates/common/flockfile/src/unix.rs

```diff
@@ -206,4 +218,19 @@ mod tests {
             FlockfileError::FromNix { .. }
         );
     }
+
+    #[test]
+    fn check_pid() {
+        let path = NamedTempFile::new().unwrap().into_temp_path().to_owned();
+        let _lockfile = Flockfile::new_lock(&path).unwrap();
+
+        let mut read_lockfile = OpenOptions::new().read(true).open(&path).unwrap();
+
+        let mut pid_string = String::new();
+        read_lockfile.read_to_string(&mut pid_string).unwrap();
+
+        let pid = Pid::from_raw(pid_string.parse().unwrap());
+
+        assert_eq!(pid, Pid::this());
+    }
 }
```

Comment:
Also, I feel it's better to add a test to check the behavior when the lock file creation is disabled.

Reply:
We agreed that no pid file is created when locking is disabled.

Reply:
ok, thanks for the clarification.